### PR TITLE
[ISSUE #4638] Optimize, translate and sync connetor documents with website

### LINKED ISSUE #4638: [Doc] Sync connetor documents with website and optimize
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Documentation Related

The documents of source/sink connectors can be optimized and synced to website for easier access of users.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

### PR DESCRIPTION
**Pull request by Pil0tXia**
Fixes #4638.

### Motivation

https://github.com/apache/eventmesh/issues/4638

### Modifications

- The documents of source/sink connectors are synced with website.
- Connectors Status sheet was re-designed.
- Translation and format optimization. The reference source for translating "sink" as "数据汇" can be found at: https://nightlies.apache.org/flink/flink-docs-release-1.18/zh/docs/learn-flink/overview/

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-connectors/eventmesh-connector-knative/README_CN.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Knative Connector 插件
```
[the remark this answers is not in the document]
Comment (xwm1992) [reply eventmesh-connectors/eventmesh-connector-knative/README_CN.md:1]: This doc seems copy from the official site: https://eventmesh.apache.org/docs/next/design-document/connect/knative-connector

**Review by xwm1992 — DISMISSED** [review]

**Comment by fabian4** [comment]
@pandaapo @xwm1992 

The Knative Connector has changed a lot and does not work with the previous way. 
- Instead of [cloudevents-player](https://knative.dev/docs/getting-started/first-source/), we can now send to the borker directlly.
- The source connector is stills in progress for the message push strategy in Knative. ( Which I will submit a pr to fix soon.)

@Pil0tXia 
So How about leave `eventmesh-connector-knative` doc blank for me to complete it in another pr.

File: eventmesh-connectors/eventmesh-connector-knative/README_CN.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-connectors/eventmesh-connector-knative/README_CN.md:1]: Yes, the Knative doc was synced with `site` repo.

**Comment by Pil0tXia** [comment]
@fabian4 I will mark knative doc as `outdated`.

**Review by fabian4 — APPROVED** [review]

File: eventmesh-connectors/README.md [hunk 2]
```diff
@@ -1,71 +1,51 @@
 # Connectors
 
-## Connector 
+>With the changes in the design of the Knative connector, this document is currently temporarily outdated.
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-connectors/README.md]: removed

**Review by xwm1992 — APPROVED** [review]

**Review by mxsm — APPROVED** [review]

**Pull request merged**